vagrant.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Latest LTS as of Nov 22, 2016
+NODE_VERSION="6.9.1"
+
+# Install new nodejs.
+_node_filename="node-v${NODE_VERSION}-linux-x64.tar.xz"
+_node_url="https://nodejs.org/dist/v${NODE_VERSION}/${_node_filename}"
```

Comment:
Oops, never mind. I realize I probably made a mistake. `nvm` does use the precompiled binaries from the official website when possible. Sorry.